torch/csrc/jit/mobile/function.cpp

```diff
@@ -99,20 +99,23 @@ bool Function::append_operator(
     // from model. We can use it to handle backward compatibility.
     if (num_specified_args &&
         num_specified_args.value() < static_cast<int64_t>(args.size())) {
-      // Sanity check at load time, to save perf at runtime
-      for (size_t i = num_specified_args.value(); i < args.size(); ++i) {
-        auto default_val = args[i].default_value();
-        TORCH_CHECK(
-            default_val.has_value(),
-            "Error happened at preparing for default values for the argument. The ",
-            i,
-            "th arguement of operator",
-            opname,
-            " does not have a specified value or default value. ");
-      }
       fn = [fn, num_specified_args, args](Stack& stack) {
-        for (size_t i = num_specified_args.value(); i < args.size(); ++i) {
-          stack.push_back(args[i].default_value());
+        c10::optional<IValue> out_arg;
+        if (!args.empty() && args.back().is_out()) {
```

Comment:
Is there always zero or one out arg? What happens if there are 2 or more out args?

Reply:
Oh I didn't know it's possible. My understanding is that out arg is an optional argument to store the output to. The output from inference can only be one. What about having a TORCH_CHECK to check only 1 out arg and we can extend it when we know there might be more than one out arg?

Reply:
I think there can be more than 1 out arg. We definitely are coding the out arg as a counter not a bool (plus it makes sense it can be > 1).

Reply:
Update to cover multiple args.